fuzzers/032-cmt-pll/generate.tcl

```diff
@@ -24,10 +19,29 @@ set_property IS_ENABLED 0 [get_drc_checks {AVAL-50}]
 set_property IS_ENABLED 0 [get_drc_checks {AVAL-53}]
 set_property IS_ENABLED 0 [get_drc_checks {REQP-126}]
 # PLL
+set_property IS_ENABLED 0 [get_drc_checks {PDRC-43}]
```

Comment:
I think we should start commenting why we disable some drcs. I know that usually it's to keep Vivado from complaining, but at least one sentence what we are disabling should be enough.